segmentation violation when kube-router is not able to contact api server

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x127f7ba]

```
goroutine 1 [running]:
github.com/cloudnativelabs/kube-router/app/watchers.StartNetworkPolicyWatcher(0xc42019a160, 0xdf8475800, 0xc4202e3860, 0x0, 0x0)
    /home/travis/gopath/src/github.com/cloudnativelabs/kube-router/app/watchers/network_policy_watcher.go:82 +0x1ca
github.com/cloudnativelabs/kube-router/app.(*KubeRouter).startApiWatchers(0xc4201f50c0, 0xc4201f50f0, 0x0)
    /home/travis/gopath/src/github.com/cloudnativelabs/kube-router/app/server.go:75 +0x261
github.com/cloudnativelabs/kube-router/app.(*KubeRouter).Run(0xc4201f50c0, 0xc4201f50c0, 0x0)
    /home/travis/gopath/src/github.com/cloudnativelabs/kube-router/app/server.go:113 +0x5d
main.main()
    /home/travis/gopath/src/github.com/cloudnativelabs/kube-router/kube-router.go:47 +0x1c6
```

--- a/app/watchers/network_policy_watcher.go
+++ b/app/watchers/network_policy_watcher.go
@@ -4,6 +4,7 @@ import (
 	"reflect"
 	"strconv"
 	"time"
+	"errors"
 
 	"github.com/cloudnativelabs/kube-router/utils"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
@@ -78,7 +79,11 @@ func StartNetworkPolicyWatcher(clientset *kubernetes.Clientset, resyncPeriod tim
 	npw.clientset = clientset
 
 	v1NetworkPolicy := true
-	v, _ := clientset.Discovery().ServerVersion()
+	v, err := clientset.Discovery().ServerVersion()
+	if err != nil {
+       return nil, errors.New("Failed to get API server version due to " + err.Error()) 
+	}
+
 	minorVer, _ := strconv.Atoi(v.Minor)
 	if v.Major == "1" && minorVer < 7 {
 		v1NetworkPolicy = false

